backend/vkt/src/main/java/fi/oph/vkt/config/security/WebSecurityConfig.java

```diff
@@ -143,20 +144,23 @@ public SecurityFilterChain filterChain(
 
   public static HttpSecurity commonConfig(final HttpSecurity http) throws Exception {
     return configCsrf(http)
-      .authorizeHttpRequests()
-      .mvcMatchers("/api/v1/clerk/**", "/virkailija/**", "/virkailija")
-      .hasRole(Constants.APP_ROLE)
-      .mvcMatchers("/", "/**")
-      .permitAll()
-      .anyRequest()
-      .authenticated()
-      .and();
+      .authorizeHttpRequests(authorize ->
+        authorize
+          .requestMatchers("/api/v1/clerk/**", "/virkailija/**", "/virkailija")
+          .hasRole(Constants.APP_ROLE)
+          .requestMatchers("/", "/**")
+          .permitAll()
+          .anyRequest()
+          .authenticated()
+      );
   }
 
   public static HttpSecurity configCsrf(final HttpSecurity http) throws Exception {
     final CookieCsrfTokenRepository csrfTokenRepository = CookieCsrfTokenRepository.withHttpOnlyFalse();
```

Comment:
Täällä vois teoriassa uudelleenjärjestellä näitä rivejä hieman, ei nyt tosin kovin tarpeellista.

```
    final CookieCsrfTokenRepository csrfTokenRepository = CookieCsrfTokenRepository.withHttpOnlyFalse();
    csrfTokenRepository.setCookieName("CSRF");
    csrfTokenRepository.setHeaderName("CSRF");

    final CsrfTokenRequestAttributeHandler requestHandler = new CsrfTokenRequestAttributeHandler();
    requestHandler.setCsrfRequestAttributeName(null);

    return http.csrf(csrf -> csrf.csrfTokenRepository(csrfTokenRepository).csrfTokenRequestHandler(requestHandler));
```